cloudlink/client.py

```diff
@@ -0,0 +1,123 @@
+from cloudlink.types import api_error_statuscode_map, statuscodes
+from os import environ
+from typing import Literal
+from urllib.parse import parse_qs, urlparse
+from websockets import WebSocketServerProtocol
+import requests
+
+class CloudlinkClient:
+	def __init__(
+		self,
+		server,
+		websocket: WebSocketServerProtocol
+	):
+		# Python moment:tm:
+		import cloudlink.server
+		
+		# Setup client
+		self.server: cloudlink.server.CloudlinkServer = server
+		self.websocket = websocket
```

Comment:
use typing.TYPE_CHECKING to import recursive-type definitions.

https://peps.python.org/pep-0484/#runtime-or-type-checking